Dockerfile

```diff
@@ -15,10 +14,15 @@ ENV PYTHONUNBUFFERED=1
 
 COPY --from=builder /app .
 
-RUN pip install --upgrade pip && \
-    pip install -r requirements.txt
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    && pip install --upgrade pip \
+    && pip install mysqlclient \
+    && pip install -r requirements.txt
```

Comment:
The `apt-get install` command should include `apt-get update` before installing packages to ensure the package list is up-to-date. Additionally, consider cleaning up the apt cache after installation to reduce the image size. You can do this by adding `&& rm -rf /var/lib/apt/lists/*` at the end of the RUN command.